README.md

```diff
@@ -1,7 +1,8 @@
 # leaflet.offline version 2.x
 
 [![npm version](https://badge.fury.io/js/leaflet.offline.svg)](https://badge.fury.io/js/leaflet.offline)
-[![Build Status](https://travis-ci.org/allartk/leaflet.offline.svg?branch=travis)](https://travis-ci.org/allartk/leaflet.offline)
+[![Build Status](https://travis-ci.org/helgasoft/leaflet.offline.png?branch=master)](https://travis-ci.org/helgasoft/leaflet.offline)
```

Comment:
could you undo this change before merge? :)

Reply:
just did.  If you need further changes, please feel free to do them -  _Allow edits from maintainers_ is enabled.